.gitignore

```diff
@@ -8,13 +8,13 @@ lightning_logs/
 .vscode/
 
 # Documentations
-docs/source/api
+docs/source-PL/api
```

Comment:
nit: I'm for `docs/source-pl` (in lowercase).

Reply:
well lets leave this for later :)